docs/agent-based-installer.md

```diff
@@ -0,0 +1,42 @@
+# Agent-based installer
+
+The assisted-service repo contains a client used by the agent-based installer. 
+The client code is located at [cmd/agentbasedinstaller](cmd/agentbasedinstaller/). 
```

Comment:
this location URL is broken. should probably start with `../cmd`

Reply:
Thanks. Fixed the location URL.